crates/web-sys/webidls/enabled/Streams.webidl

```diff
@@ -6,19 +6,21 @@
  *
  * The origin of this IDL file is
  * https://streams.spec.whatwg.org/#idl-index
+ * `[Throws]` attributes copied from Mozilla's webidl tree as of Aug. 31, 2022
+ * https://hg.mozilla.org/mozilla-central/file/tip/dom/webidl/
  */
 
 [Exposed=*, Transferable]
 interface ReadableStream {
-  constructor(optional object underlyingSource, optional QueuingStrategy strategy = {});
+  [Throws] constructor(optional object underlyingSource, optional QueuingStrategy strategy = {});
 
   readonly attribute boolean locked;
 
   Promise<undefined> cancel(optional any reason);
-  ReadableStreamReader getReader(optional ReadableStreamGetReaderOptions options = {});
-  ReadableStream pipeThrough(ReadableWritablePair transform, optional StreamPipeOptions options = {});
+  [Throws] ReadableStreamReader getReader(optional ReadableStreamGetReaderOptions options = {});
+  [Throws] ReadableStream pipeThrough(ReadableWritablePair transform, optional StreamPipeOptions options = {});
   Promise<undefined> pipeTo(WritableStream destination, optional StreamPipeOptions options = {});
-  sequence<ReadableStream> tee();
+  [Throws] sequence<ReadableStream> tee();
```

Comment:
Although this is correct, it's a breaking change, since `ReadableStream` is already stable with methods which don't return `Result`. So, I think we'll have to leave these as being incorrectly infallible.

This isn't a problem for the other types because they were all previously unstable.